lib/ruby_indexer/test/constant_test.rb

```diff
@@ -0,0 +1,94 @@
+# typed: true
+# frozen_string_literal: true
+
+require_relative "test_case"
+
+module RubyIndexer
+  class ConstantTest < TestCase
```

Comment:
Let's also add a test for `var::CONSTANT` just to make sure we don't try to index those.